packages/experimental/Avatar/macos/MSFAvatarViewManager.m

```diff
@@ -0,0 +1,34 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+
+@import FluentUI;
+
+@interface RCT_EXTERN_MODULE(MSFAvatarViewManager, RCTViewManager)
+
+RCT_REMAP_VIEW_PROPERTY(primaryText, contactName, NSString)
+
+RCT_REMAP_VIEW_PROPERTY(secondaryText, contactEmail, NSString)
+
+RCT_REMAP_VIEW_PROPERTY(imageSource, contactImage, UIImage)
+
+RCT_REMAP_VIEW_PROPERTY(backgroundColor, avatarBackgroundColor, UIColor)
```

Comment:
Fun fact: even though this is macOS, you need to set these as `UIImage` and `UIColor` for it to work properly

Reply:
can we add comments on this? i cringe a little

Reply:
Yes.. though lots of controls will have things that maps to colors and images (like in Amanda's change). Is it still worth adding a comment if I have to add it everywhere?

Reply:
Does this just work because UIImage and UIColor are macros defined to the correct platform types?

Reply:
If so, react-native-macos defines various RCT* wrappers around these native types that we could try using instead

Reply:
As in, specify something like RCTUIColor instead? I'll try that.
It seems that UIColor and UIImage map to their corresponding RCTConvert macros, which contain the correct macOS implementation. Both and and @chiuam had problems setting it to just NSColor.

Reply:
Yea we need to change that at some point, but probably fine for now as that's out of scope for this review